code/modules/antagonists/blob/structures/factory.dm

```diff
@@ -18,6 +19,11 @@
 	///Used in blob/powers.dm, checks if it's already trying to spawn a blobbernaut to prevent issues.
 	var/is_creating_blobbernaut = FALSE
 
+/datum/armor/special_factory
+	fire = 80
+	acid = 70
+	laser = 25
```

Comment:
Maybe subtype `/datum/armor/structure_blob` for this?

Reply:
This is intentional, I don't want factories and resource nodes to have the full 50 armor.